bin/scpipe/lib/sc/pipe.rb

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env ruby
+#
+# SC:Pipe derived from sclang_pipe (SCVIM Package)
+# Copyright 2007 Alex Norman under GPL
+#
+# modified 2010 stephen lumenta
+# modified 2012 José Fernández Ramos
+
+require 'fileutils'
+require 'singleton'
+require 'tmpdir'
+
+module SC
+
+  @@sclang_path = `which sclang`
+
+  def self.sclang_path
+    if @@sclang_path.empty?
+      if File.exists?("/Applications/SuperCollider.app/Contents/Resources/sclang")
```

Comment:
I think there are still some people on 3.6.6, so it might be a good idea to check for both paths! I don't have a strong opinion one way or the other, but it should at the very least check the newer path.